src/server/plugins/engine/pageControllers/SummaryPageController.ts

```diff
@@ -119,13 +119,22 @@ export class SummaryPageController extends PageController {
         }
       }
 
+      const { params } = request
+      const { slug } = params as { slug: string }
```

Comment:
Mind if we hold off using `as` for now?

You'll have `params.slug` working as expected after https://github.com/DEFRA/forms-runner/pull/469/commits/4945bb5043312d3b2ee83af25a6ce8465eddef85 is merged